src/replaykit.cs

```diff
@@ -259,5 +259,9 @@ interface RPBroadcastSampleHandler {
 
 		[Export ("processSampleBuffer:withType:")]
 		void ProcessSampleBuffer (CMSampleBuffer sampleBuffer, RPSampleBufferType sampleBufferType);
+
+		[iOS (10,2)][TV (10,1)]
+		[Export ("finishBroadcastWithError:")]
+		void FinishBroadcast (NSError error);
```

Comment:
No, `- (void)finishBroadcastWithError:(NSError *)error;`. Header description suggest you're sopping the broadcast so it's likely up to the caller to say why (using an `NSError`)